.buildkite/scripts/shared.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
```

Comment:
No `set -euo pipefail`?

Reply:
This basically a library script, and doesn't actually run anything